satellite/metainfo/metainfo.go

```diff
@@ -957,6 +985,7 @@ func convertProtoToBucket(req *pb.BucketCreateRequest, projectID uuid.UUID) (buc
 	defaultRS := req.GetDefaultRedundancyScheme()
 	defaultEP := req.GetDefaultEncryptionParameters()
 
+	// TODO: resolve partner id
```

Comment:
I'm not clear what has to be resolved here, could you briefly explain me?

Reply:
I'm not 100% sure but I guess with partnerID we can resolve this to `PartnerInfo`. See satellite/rewards/partners.go and `DefaultPartners`.